### PR TITLE
Created and Added all the languages that support Emoji

### PR DESCRIPTION
**Pull request by SethiShreya**
Signed off by Shreya Sethi
shreyasethi47@gmail.com

This PR addresses issue #397 
- added a JSON file which includes all the languages that support emoji datatype to compare to
- created a centralized generate_emoji_keywords.py in scribe_data/unicode which takes a language, and output directory from main.py to generate emoji
- Configure other files to integrate it well

@andrewtavis @Ekikereabasi-Nk please review it and give your feedback :)

### CODE REVIEW

**Comment by Ekikereabasi-Nk** [comment]
@SethiShreya nice work

File: src/scribe_data/unicode/generate_emoji_keywords.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+def generate_emoji(language, output_dir: str = None):
+    print(f"Got the language {language} for emoji generation")
+
+    # check if this language is supported
+    with open(SUPPORTED_LANGUAGE_FILE, "r", encoding="utf-8") as file:
+        languages = json.load(file)
+    # Check if the language code exists in the dictionary
+    for code, name in languages.items():
+        if name == language:
+            print(f"Emoji Generation for language : {language} is supported")
```
Comment (axif0) [inline src/scribe_data/unicode/generate_emoji_keywords.py:42]: For lower and capitalization we need to dynamically handle it

File: src/scribe_data/unicode/generate_emoji_keywords.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    export_dir.mkdir(parents=True, exist_ok=True)
+
+    if emoji_keywords_dict := gen_emoji_lexicon(
+        language=language,
+        emojis_per_keyword=EMOJI_KEYWORDS_DICT,
+    ):export_formatted_data(
+            file_path=output_dir,
+            formatted_data=emoji_keywords_dict,
+            query_data_in_use=True,
+            language=language,
```
Comment (axif0) [inline src/scribe_data/unicode/generate_emoji_keywords.py:59]: We should use language.capitalize(), therefore no `No such file or directory: 'scribe_data_json_export/english/emoji_keywords.json'` will cause. In file data-type should be `'{user_given_directory}/English/emoji_keywords.json'`

**Comment by axif0** [comment]
cc: @mhmohona Can you please review in a new environment if you have free time in hand, as you got windows OS? Also, by applying the upper issues?

**Review by andrewtavis — APPROVED** [review]
This is really really great, @SethiShreya :) I removed the JSON so we can base this directly off of the CLDR data as I think that's more sustainable. I'm getting the PyICU error for it not being installed as well, but I think we can close that with @axif0's PR or a combination of that one and your other one.

**Pull request merged**